tests/nightwatch/login.js

```diff
@@ -0,0 +1,17 @@
+describe('Login into nextcloud end-to-end test', function() {
```

Comment:
Can you replace `'Login into nextcloud end-to-end test'` by `Chek if the workspace app is present', please ?